CHANGELOG.rst

```diff
@@ -1,8 +1,8 @@
 Changelog
 =========
 
-in development
```

Comment:
May need that string for future `in development` changelog, otherwise contributors less likely to include them